### DISCUSSION TITLE
About the bug of tool json-to-generator

### DISCUSSION THREAD

**Discussion by mimimiaomiao1** [he3-app/he3-feedback#308] [Bug Report]
### Version
json-to-generator 1.0.7

### Environment
Mac v2.0.2

### What is expected?

1. Add a button with the function of creating a simple JSON
**LIKE**

{
	"animals": {
		"dog": [{
				"name": "Rufus",
				"breed": "labrador",
				"count": 1,
				"twoFeet": false
			},
			{
				"name": "Marty",
				"breed": "whippet",
				"count": 1,
				"twoFeet": false
			}
		],
		"cat": {
			"name": "Matilda"
		}
	}
}

**Comment by lyzhang1999** [comment]
https://github.com/he3-app/he3-feedback/issues/309
